src/Content/Backend/Solution/Monaco.Template.Backend.Application.Tests/Features/Product/CreateProductValidatorTests.cs

```diff
@@ -110,10 +105,7 @@ public async Task DescriptionDoesNotGenerateErrorWhenValid()
 	{
 		_dbContextMock.CreateAndSetupDbSetMock(new List<Domain.Model.Product>());
 
-		var command = Command with
-					  {
-						  Description = new string(It.IsAny<char>(), 100)
-					  };
+		var command = Command with { Description = new string(It.IsAny<char>(), 100) };
```

Comment:
This should be checking if a length of 500 is valid.

Perhaps we should be advocating for declaring constants in the domain class for these constraints rather than using 'magic values' to ensure consistency and easier maintainability throughout the app? For example, the Product Description Max Length constraint is currently typed out separately in the following files (sometimes more than once in the same file e.g. validator tests):
- Domain > `Product`
- Domain Tests > `ProductTest`
- Infrastructure > `ProductEntityConfiguration`
- Application > Features > Product > Create > `Validator`
- Application > Features > Product > Edit > `Validator`
- Application Tests > Features > Product > `CreateProductValidatorTests`
- Application Tests > Features > Product > `EditProductValidatorTests`

To solve this we can just add a constant such as the following to the domain class:

```csharp
public class Product : Entity
{
   public const int TITLE_MAX_LENGTH = 100;
   public const int DESCRIPTION_MAX_LENGTH = 500;
   ... // rest of class
}
```

then elsewhere we can use simply such as in the above case:

```csharp
var command = Command with { Description = new string(It.IsAny<char>(), Domain.Model.Product.DESCRIPTION_MAX_LENGTH) };
```

What do you think?

Reply:
Yeah, it's been something I've been thinking about while I was updating these validations and tests but wasn't sure if introducing it in this PR, but I guess is as good time as any to do it 😅 
Guess I'll include it here to get it all clean and nice at once :)

Reply:
Done.